NEWS.md

```diff
@@ -1,60 +1,55 @@
-3.23.1
+3.24.1
 ======
 
+## Highlights
 
-## Features
-
- * The `redis()` destination now handles any number of command parameters.
-   (#2816)
-
- * The `format()` option of `date-parser()` supports lists. From now on, a
-   single `date-parser()` instance is able to process different date formats,
-   making it easy to catch on when some programs change the way they log
-   information. (#2779)
+ * Add a new template function called `format-flat-json()`, which generates
+   flattened json output. This is useful for destinations, where the json
+   parsing does not handle nested json format. (#2890)
+ * Add ISO 8601 compliant week numbering. Use it with the `${ISOWEEK}` macro
+   and the timestamp prefixes `P_, S_, R_, C_` (#2878)
```

Comment:
... and all its variants S_ISOWEEK, R_ISOWEEK and C_ISOWEEK 

The P timestamp is an internal hack for SSB and hasn't been documented IIRC.

Reply:
Thanks, fixed!